CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ HunterGate(
 
 # project configuration
 set(PROJECT_NAME_STR pandora)
-project(${PROJECT_NAME_STR} VERSION "0.11.0" LANGUAGES C CXX)
+project(${PROJECT_NAME_STR} VERSION "0.10.0" LANGUAGES C CXX)
```

Comment:
Just wondering, is it possible to make the version 0.10.0-alpha.1 here? I tried looking at the CMake docs and it wasn't super clear...

Reply:
I've tried this before and was unable to do it, but will retry just in case

Reply:
I didn't manage to do it, there is a 5-year open issue in CMake git repo asking for this feature, but hasn't been finished yet: https://gitlab.kitware.com/cmake/cmake/-/issues/16716 . So I am keeping the version as is I guess...

Reply:
Oh, we can actually use an extra number for the tweak version, so won't be able to set `0.10.0-alpha.1` but can set `0.10.0.1`.

Reply:
Just did this in https://github.com/rmcolq/pandora/pull/318/commits/524c7ae755e1a711c717ec3a9bf2aa3c5292399f , hope is ok. I am then merging this PR. If there is a problem, please tell me in slack or raise an issue.